docs/deploy_streamlit_app.md

```diff
@@ -84,11 +84,26 @@ Streamlit sharing is completely free and is meant to get you started with sharin
 
 ### Access
 
-- Apps are always visible to the entire world.
+- Apps are visible to the entire world.
 - You can only deploy apps that are in a public GitHub repo.
 - Your source code must live in Github. We're looking to expand to other Git hosts soon.
 - Everyone with push access to your repo is automatically a maintainer of the app.
 
+### Long Term
```

Comment:
"Long Term" feels like the wrong heading here. I would be much more direct with "App Hibernation", "Automatic Spindown" or so that even if people don't read the text, the  idea will be in their head

Reply:
I agree! We were having trouble with a good header for this section, I changed this to use the "App Hibernation" you suggested